make.bat

```diff
@@ -6,7 +6,8 @@ set _EXITCODE=0
 REM If no target is provided, default to test.
 if [%1]==[] goto test
 
-set _TARGETS=bin,dev,generate,test,testacc,testrace,vet
+set _TARGETS=bin,bootstrap,dev,generate,test,testacc,testrace,vet
+set _EXTERNAL_TOOLS=github.com/mitchellh/gox,github.com/kardianos/govendor,github.com/client9/misspell/cmd/misspell
```

Comment:
misspell shouldn't be necessary as a global thing for people to install

Reply:
I agree about misspell... I replicated what was in `Makefile`.

```
$ grep misspell Makefile
        github.com/client9/misspell/cmd/misspell
        @misspell -error -source=text website/source
```

Your call whether we should keep it or not!

Reply:
It's only used for the website target, so not needed for normal builds.